src/Configuration/Config.Azure.KeyVault.Secrets/src/Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Azure KeyVault configuration provider implementation for Microsoft.Extensions.Configuration.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>$(PackageTags);azure;keyvault</PackageTags>
+    <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+    <AdditionalGenApiCmdOptions>$(AdditionalGenApiCmdOptions) --global</AdditionalGenApiCmdOptions>
```

Comment:
Why does this project have an associated ref/ project? Is it part of the shared Fx now?